src/lines.jl

```diff
@@ -10,280 +10,293 @@ module Lines
     function Next()::Int
         return Lib.Lines_Get_Next()
     end
 
-    """Normal ampere rating of Line."""
+    """Normal ampere rating of Line. (Getter)"""
     function NormAmps()::Float64
         return Lib.Lines_Get_NormAmps()
     end
 
-    """Normal ampere rating of Line."""
+    """Normal ampere rating of Line. (Setter)"""
     function NormAmps(Value::Float64)
         Lib.Lines_Set_NormAmps(Value)
     end
 
-    """(read-only) Number of customers on this line section."""
+    """Number of customers on this line section."""
     function NumCust()::Int
         return Lib.Lines_Get_NumCust()
     end
 
-    """(read-only) Sets Parent of the active Line to be the active line. Returns 0 if no parent or action fails."""
+    """Sets Parent of the active Line to be the active line. Returns 0 if no parent or action fails."""
     function Parent()::Int
         return Lib.Lines_Get_Parent()
     end
 
-    """Number of Phases, this Line element."""
+    """Number of Phases, this Line element. (Getter)"""
     function Phases()::Int
         return Lib.Lines_Get_Phases()
     end
 
-    """Number of Phases, this Line element."""
+    """Number of Phases, this Line element. (Setter)"""
     function Phases(Value::Int)
         Lib.Lines_Set_Phases(Value)
     end
 
-    """Zero Sequence resistance, ohms per unit length."""
+    """Zero Sequence resistance, ohms per unit length. (Getter)"""
     function R0()::Float64
         return Lib.Lines_Get_R0()
     end
 
-    """Zero Sequence resistance, ohms per unit length."""
+    """Zero Sequence resistance, ohms per unit length. (Setter)"""
     function R0(Value::Float64)
         Lib.Lines_Set_R0(Value)
     end
 
-    """Positive Sequence resistance, ohms per unit length."""
+    """Positive Sequence resistance, ohms per unit length. (Getter)"""
     function R1()::Float64
         return Lib.Lines_Get_R1()
     end
 
-    """Positive Sequence resistance, ohms per unit length."""
+    """Positive Sequence resistance, ohms per unit length. (Setter)"""
     function R1(Value::Float64)
         Lib.Lines_Set_R1(Value)
     end
 
-    """Earth return resistance value used to compute line impedances at power frequency"""
+    """Earth return resistance value used to compute line impedances at power frequency (Getter)"""
     function Rg()::Float64
         return Lib.Lines_Get_Rg()
     end
 
-    """Earth return resistance value used to compute line impedances at power frequency"""
+    """Earth return resistance value used to compute line impedances at power frequency (Setter)"""
     function Rg(Value::Float64)
         Lib.Lines_Set_Rg(Value)
     end
 
-    """Earth Resistivity, m-ohms"""
+    """Earth Resistivity, m-ohms (Getter)"""
     function Rho()::Float64
         return Lib.Lines_Get_Rho()
     end
 
-    """Earth Resistivity, m-ohms"""
+    """Earth Resistivity, m-ohms (Setter)"""
```

Comment:
"m-ohms" -> "ohm-m"  (so it doesn't get confused with milliohms.

Reply:
Fixed in cc7d20d.